src/reducers/comments.tsx

```diff
@@ -156,6 +170,73 @@ export const selectVersionHasComments = ({
   return Object.keys(comments.byId).length > 0;
 };
 
+export const fetchAndLoadComments = ({
+  _getComments = getComments,
+  addonId,
+  versionId,
+}: {
+  _getComments?: typeof getComments;
+  addonId: number;
+  versionId: number;
+}): ThunkActionCreator => {
+  return async (dispatch, getState) => {
+    const { api: apiState, comments } = getState();
+
+    if (comments.isLoading) {
+      return;
```

Comment:
Would it be worth logging a debug message here?

Reply:
I usually prefer it that way but in the current implementation (in a forthcoming patch), the debug message would get printed once for each line in the file. So, for a 1000 line file it would be printed 1000 times. This makes me hesitant to log anything.